app/src/ui/merge-branch/merge.tsx

```diff
@@ -38,6 +38,11 @@ interface IMergeProps {
    */
   readonly recentBranches: ReadonlyArray<Branch>
 
+  /**
+   * The branch to select when the merge dialog is opened
+   */
+  readonly preselectedBranch?: Branch
```

Comment:
The convention we've had in a couple of other places is to use `initial` as a prefix

e.g:

 - `initialPath` in our `CreateRepository` dialog
 - `initialURL` in our `CloneRepository` dialog
 - `initialName` in our `CreateBranch` dialog 

Maybe this should align with that?